build/Dockerfile

```diff
@@ -72,5 +74,6 @@ RUN chown -R steam:steam /usr/local/lib/dsta/ \
 EXPOSE 10999/udp
 VOLUME ["/var/lib/dsta/config"]
 
-WORKDIR /opt/dst/bin/
+WORKDIR /var/lib/dsta/config
```

Comment:
We don't need to be in the /opt/dst/bin/ to run the server now. `dedicated_server_nullrenderer` script do the trick. I think that the more interesant directory in all the container is that. We can even remove the `WORKDIR` but I think that it's good because it help to discovery the dir.